### PR TITLE
[Torch] how to add torch converter

### PR DESCRIPTION
**Pull request by fortianyou**

### CODE REVIEW

**Review by linearhit — APPROVED** [review]
LGTM

File: docs/developers/torch_add_a_new_converter.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# How To Add a New Torch Op Converter
+
+## 1. Get the Operator's Function Schema
```
Comment (Yancey1989) [inline docs/developers/torch_add_a_new_converter.md:3]: As the comment: https://github.com/alibaba/BladeDISC/pull/17#discussion_r775353050 
please remove the number on the title.

File: docs/developers/torch_add_a_new_converter.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+You can find aten native node schema definitions at
+[ATen/native](https://github.com/pytorch/pytorch/tree/master/aten/src/ATen/native);
+The TorchBlade also provides a tool function `node_schema_str` that returns the
```
Comment (Yancey1989) [inline docs/developers/torch_add_a_new_converter.md:7]: reference to the definition of source code?

File: docs/developers/torch_add_a_new_converter.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+@torch.jit.script
+def add(x, y):
+    return x + y
+
+print(add.graph)
+for n in add.graph.nodes():
+    print(tools.node_schema_str(n))
+```
+
+```
```
Comment (Yancey1989) [inline docs/developers/torch_add_a_new_converter.md:23]: can use ``` text

File: docs/developers/torch_add_a_new_converter.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      %y.1 : Tensor):
+  %4 : int = prim::Constant[value=1]()
+  %5 : Tensor = aten::add(%x.1, %y.1, %4)
+  return (%5)
+
+
+aten::add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> (Tensor)
+```
+
+## 2. Create an Operator Converter
```
Comment (Yancey1989) [inline docs/developers/torch_add_a_new_converter.md:34]: mix `Op` and `Operator`, maybe we can be consistent.

**Pull request merged**